### PR TITLE
New rocAL python API

### PR DESCRIPTION
**Pull request by swetha097**
1. Augmentation App
2. coco pipeline
3. caffe, caffe2 pipelines for classification & detection
4. tf classification & detection pipeline
5. python rali unit test

### CODE REVIEW

File: rocAL/rocAL/source/master_graph.cpp [hunk 1]
```diff
@@ -17,7 +17,7 @@ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
 AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-THE SOFTWARE.
+THE SOFTWARE.lround
```
Comment (LakshmiKumar23) [inline rocAL/rocAL/source/master_graph.cpp]: @swetha097 Can you check what this is. Seems like a typo?

File: rocAL/rocAL/source/rali_api_data_loaders.cpp [hunk 2]
```diff
@@ -655,6 +655,7 @@ raliJpegCOCOFileSourceSingleShard(
         unsigned max_width,
         unsigned max_height)
 {
+    std::cout<<"\n COMES TO COCOFILESOURCESINGLESHARD";
```
Comment (LakshmiKumar23) [inline rocAL/rocAL/source/rali_api_data_loaders.cpp]: @swetha097 Do we need the debug statements? Same above?

**Review by LakshmiKumar23 — CHANGES_REQUESTED** [review]
@swetha097 I also see a couple of empty files. Are they needed?

File: rocAL/rocAL/source/rali_api_augmentation.cpp [hunk 3]
```diff
@@ -478,6 +478,10 @@ raliBrightness(
         RaliFloatParam p_alpha,
         RaliFloatParam p_beta)
 {
+    std::cerr<<"\n**************************************************************************";
```
Comment (rrawther) [inline rocAL/rocAL/source/rali_api_augmentation.cpp]: I think this should be removed

**Review by rrawther — COMMENTED** [review]
@swetha097  need to review this

File: rocAL/rocAL/source/rali_api_augmentation.cpp (hunk 3 above)
Comment (swetha097) [reply rocAL/rocAL/source/rali_api_augmentation.cpp]: Done

File: rocAL/rocAL/source/rali_api_data_loaders.cpp (hunk 2 above)
Comment (swetha097) [reply rocAL/rocAL/source/rali_api_data_loaders.cpp]: Removed the debug statement

File: rocAL/rocAL/source/master_graph.cpp (hunk 1 above)
Comment (swetha097) [reply rocAL/rocAL/source/master_graph.cpp]: I think its a typo Lakshmi.
Removed it

**Pull request closed without merging**